README.md

```diff
@@ -148,6 +148,7 @@ Prometheus uses file watches and all changes to the json file are applied immedi
 | check-single-keys       | REDIS_EXPORTER_CHECK_SINGLE_KEYS       | Comma separated list of keys to export value and length/size, eg: `db3=user_count` will export key `user_count` from db `3`. db defaults to `0` if omitted.  The keys specified with this flag will be looked up directly without any glob pattern matching.  Use this option if you don't need glob pattern matching;  it is faster than `check-keys`.                                                                                                                                                                                           |
 | check-streams           | REDIS_EXPORTER_CHECK_STREAMS           | Comma separated list of stream-patterns to export info about streams, groups and consumers. Syntax is the same as `check-keys`.                                                                                                                                                                                                                                                                                                                                                                                                                   |
 | check-single-streams    | REDIS_EXPORTER_CHECK_SINGLE_STREAMS    | Comma separated list of streams to export info about streams, groups and consumers. The streams specified with this flag will be looked up directly without any glob pattern matching.  Use this option if you don't need glob pattern matching;  it is faster than `check-streams`.                                                                                                                                                                                                                                                              |
+| streams-exclude-consumer    | REDIS_EXPORTER_STREAMS_EXCLUDE_CONSUMER    | Don't collect per consumer metrics for streams (decreases amount of metrics and cardinality).                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
REDIS_EXPORTER_STREAMS_EXCLUDE_CONSUMER -> REDIS_EXPORTER_STREAMS_EXCLUDE_CONSUMER_METRICS ?

Reply:
Yep, sure.

My only concern was that it would be quite long, but not too much I think)

Reply:
Done, committed changes.

Reply:
Apparently tests are failing, but I don't see exact reason.

In the end of output only 

```log
FAIL
coverage: 90.0% of statements
FAIL	github.com/oliver006/redis_exporter/exporter	22.273s
FAIL
```

```log

```